### PR TITLE
Enable ezplatform-http-cache-fastly

### PR DESCRIPTION
**Pull request by vidarl**
This branch enables ezplatform-http-cache-fastly in ee

This PR should be rebased once https://github.com/ezsystems/ezplatform/pull/231 is merged to ezplatform-ee

### CODE REVIEW

**Comment by vidarl** [comment]
@andrerom, @bdunogier, @alongosz : review ping

File: composer.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
         "ezsystems/date-based-publisher": "^1.0@dev",
         "ezsystems/content-on-the-fly-prototype": "~0.1.7",
         "ezsystems/ezplatform-multi-file-upload": "^0.1.0@dev",
+        "ezsystems/ezplatform-http-cache-fastly": "dev-fastly-driver",
```
Comment (andrerom) [inline composer.json]: should be updated before merge I guess

Comment (vidarl) [reply composer.json]: Yes, but I don't know what we'll call the version yet
1.0.0-beta1?

File: .env [hunk 2]
```diff
@@ -29,6 +29,12 @@ WEB_HOST=web
 SELENIUM_HOST=selenium
 
 # Enable recommendations by setting valid id, key and uri
+# In order for this to work you also need to have the EzSystemsRecommendationBundle installed
```
Comment (andrerom) [inline .env]: this should have been in ezplatform to avoid conflicts

File: app/config/ezplatform.yml [hunk 3]
```diff
@@ -37,6 +38,9 @@ ezpublish:
               # As we by default enable EzSystemsPlatformHttpCacheBundle which is designed to expire all affected cache
               # on changes, and as error / redirects now have separate ttl, we easier allow ttl to be greatly increased
               default_ttl: '%httpcache_default_ttl%'
-            # HttpCache purge server(s) setting, eg Varnish, for when ezpublish.http_cache.purge_type is set to 'http'.
```
Comment (andrerom) [inline app/config/ezplatform.yml]: maybe skip this diff to avoid conflicts on future merges

File: app/config/ezplatform.yml [hunk 4]
```diff
@@ -37,6 +38,9 @@ ezpublish:
...
             http_cache:
+                # HttpCache purge server(s) setting, eg Varnish, for when ezpublish.http_cache.purge_type is set to 'http' or 'fastly'.
                 purge_servers: ['%purge_server%']
+                fastly:
+                    service_id: '%fastly_service_id%'
+                    key: '%fastly_key%'
```
Comment (andrerom) [inline app/config/ezplatform.yml]: new line

File: app/config/ezplatform.yml (hunk 3 above)
Comment (vidarl) [reply app/config/ezplatform.yml]: ok, removed

File: .env (hunk 2 above)
Comment (vidarl) [reply .env]: ok, removed

File: app/config/ezplatform.yml (hunk 4 above)
Comment (vidarl) [reply app/config/ezplatform.yml]: fixed

**Pull request merged**